### PR TITLE
Adding courses category

### PR DESCRIPTION
**Pull request by Edocampa**
## This PR

- Adding courses from udemy
- Associate skills to this courses

But when I try to execute a query, there are no results. I think the new data hasn't been correctly inserted into Virtuoso.

### CODE REVIEW

File: vocabularies/courses.ttl [hunk 1]
```diff
@@ -17,20 +17,78 @@ ex:course001 a schema:Course ;
...
     dcterms:description "This course provides a comprehensive guide to mastering Oracle SQL, covering everything from the basics to advanced topics."@en ;
     schema:url <https://www.udemy.com/course/the-complete-oracle-sql-certification-course/> ;
-    schema:teaches <http://data.europa.eu/esco/skill/598de5b0-5b58-4ea7-8058-a4bc4d18c742>,   # SQL
-                    <http://data.europa.eu/esco/skill/ab1e97ed-2319-4293-a8b7-072d2648822f> . # Database Management
+    schema:teaches  <http://data.europa.eu/esco/skill/598de5b0-5b58-4ea7-8058-a4bc4d18c742>, # SQL
+                    <http://data.europa.eu/esco/skill/3af687cb-8c7a-46ea-a730-32dbaed040c6>, # Oracle Data Integrator
+                    <http://data.europa.eu/esco/skill/e2fb1ee4-fbf9-4690-b677-3e6ff7cfdc48>, # test ICT queries
+                    <http://data.europa.eu/esco/skill/de9f85ba-e77f-48fd-8c66-f5ebaf32d655>, # Oracle Relational Database
+                    <http://data.europa.eu/esco/skill/ec85cc63-4e24-4631-bf92-8789db2605c0>, # maintain database security
+                    <http://data.europa.eu/esco/skill/70de94f5-9575-4edd-bca7-c797b023b9d6>, # Oracle Weblogic
```
Comment (ioggstream) [inline vocabularies/courses.ttl:25]: Please, remove all unrelated Oracle products, such as Weblogic and Oracle Warehouse Builder

File: vocabularies/courses.ttl [hunk 2]
```diff
@@ -17,20 +17,78 @@ ex:course001 a schema:Course ;
...
 
 ex:course002 a schema:Course ;
     rdfs:label "Ansible for the Absolute Beginner - Hands-On - DevOps"@en ;
     dcterms:description "This course introduces Ansible to the absolute beginner in DevOps. Practice Ansible with coding exercises in browser."@en ;
     schema:url <https://www.udemy.com/course/learn-ansible/> ;
-    schema:teaches <http://data.europa.eu/esco/skill/6f8a40d6-f9ce-43ec-a72f-d4213a53f3ed>
-.
+    schema:teaches <http://data.europa.eu/esco/skill/6f8a40d6-f9ce-43ec-a72f-d4213a53f3ed>, # Ansible
+                   <http://data.europa.eu/esco/skill/f0de4973-0a70-4644-8fd4-3a97080476f4>, # DevOps
+                   <http://data.europa.eu/esco/skill/6b3afb82-c4ea-4f14-be54-a757fb762663>, # Vagrant
```
Comment (ioggstream) [inline vocabularies/courses.ttl:41]: Not sure Vagrant is in scope. Please, check.

**Comment by ioggstream** [comment]
@Edocampa can you find the entries, now?

**Comment by Edocampa** [comment]
Yes, I can find the entries now. The datas appears to be correctly loaded and is accessible.

**Comment by ioggstream** [comment]
@Edocampa merged. Restart from https://github.com/par-tec/esco-playground/tree/ioggstream-26-bis

**Pull request merged**